internal/metrics/metrics.go

```diff
@@ -12,17 +12,24 @@ import (
 
 const (
 	DefaultPort              = 9090
-	metricsNamespace         = "satsuma"
-	metricsSubsystem         = "node_gateway"
+	metricsNamespace         = "node_gateway"
 	defaultReadHeaderTimeout = 10 * time.Second
+
+	// Metric labels
```

Comment:
Might be helpful to somehow namespace these as blockHeightCheckErrorTypes

Reply:
`HTTPInit` and `HTTPRequest` apply to all the checks, where only the WS ones apply to block height. I'll add more comments to clarify their usage.